SPH_SIMULATIONS.md

```diff
@@ -0,0 +1,82 @@
+## Running an SPH Simulation
+
+This section assumes users know how to configure a SplishSplash and/or DualSPHysics
+simulation, in which case our **Inductiva API client** provides a simple interface to run
+those simulations at scale. Users are, in any case, invited to consult the relevant documentation
+for each of those simulation engines (links
+[here](https://github.com/InteractiveComputerGraphics/SPlisHSPlasH) and
+[here](https://github.com/DualSPHysics/DualSPHysics), respectively). 
+
+Let's demonstrate how to use two SPH simulators with **Inductiva API**.
+
+### SplishSplash
+
+SplishSplash is simple to run and the simulator is simply configured via a few configuration files.
+The required one is a `.json` file that contains the parameters of the simulation.
+The others are extra input files that determine the geometry and properties of the
+objects and fluids in the simulation.
+
+```python
+import inductiva
+
+input_dir = inductiva.utils.files.download_from_url(
+    "https://storage.googleapis.com/inductiva-api-demo-files/"
+    "splishsplash-input-example.zip", unzip=True)
+
+# Set simulation input directory
+splishsplash = inductiva.simulators.SplishSplash()
+
+task = splishsplash.run(input_dir=input_dir,
+                        sim_config_filename="config.json")
+
+# Wait for the simulation to finish
+task.wait()
+task.download_outputs()
+```
+
+As soon you launch the simulation with this script, you will receive the following
+logs telling how the simulation is going:
+
+<img src="assets/simulation_logs.png" alt="Simulation Logs">
+
+
+### DualSPHysics
```

Comment:
let's do something similar for this case.
Get all input files into a local folder `sph_demos/simple_dualsphysics` and generate a descritption of what the sim is doing.